.travis.yml

```diff
@@ -1,29 +1,71 @@
 language: cpp
-os:
-- linux
-- osx
-compiler:
-- clang
-- gcc
+
+matrix:
```

Comment:
I don't see a lot of value in gcc. So, I intentionally left it out to keep the matrix tight. If you feel differently, then sure, add an entry to run gcc on linux.

wrt to iojs, feel free to add, but node v0.12 is already covered.